.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
This file is not necessary, the repo has `.eslintrc` file already. Please remove